content/posts/jellyfin-10-6-0.md

```diff
@@ -0,0 +1,114 @@
+
+Thankfully, this is now behind us, as [MrTimscampi](https://github.com/MrTimscampi) worked on improving the way we build the web client by using Gulp to perform various tasks necessary for building our current code structure with modern tools. ([jf-web#862](https://github.com/jellyfin/jellyfin-web/pull/862)). This allows us to use bleeding edge JavaScript thanks to Babel, but also simplifies greatly our support for legacy clients (Most notably early WebOS and Tizen versions).
+
+Among the benefits of this move to Gulp for building the client, we have started moving away from [RequireJS](https://requirejs.org/) and towards using standard [EcmaScript Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules), thanks to [Camc314](https://github.com/Camc314), [cromefire](https://github.com/cromefire/), [Delgan](https://github.com/delgan), [dkanada](https://github.com/dkanada/), [grafixeyehero](https://github.com/grafixeyehero/), [MrTimscampi](https://github.com/MrTimscampi), and [sarab97](https://github.com/sarab97/).
+
+This massive change, once it is complete, will allow us to simplify the web client build process, which will in turn allow us to start our migration to [Vue](https://vuejs.org/) and significantly clean up our code. This should also bring some noticeable performance improvements to the web client down the line, as we tackle technical debt and remove deprecated practices and libraries from the code.
+
+# Server performance, bug fixes and better networking
+
+Our resident C# performance wizard, [Bond-009](https://github.com/Bond-009) continues his quest to rid the server source of warnings and generally improve server performance, fixing multiple bugs in the process.
+
+[mark-monteiro](https://github.com/mark-monteiro) worked on multiple fixes for 10.6, including reworking the network settings accessible through the administration dashboard ([jf#2774](https://github.com/jellyfin/jellyfin/pull/2774), [jf-web#1140](https://github.com/jellyfin/jellyfin-web/pull/1140)).
+
+Team member [nyanmisaka](https://github.com/nyanmisaka), with some help from [artiume](https://github.com/artiume/), brings a number of improvements to transcoding ([jf#2809](https://github.com/jellyfin/jellyfin/pull/2809), [jf-web#1046](https://github.com/jellyfin/jellyfin-web/pull/1046), [jf#2821](https://github.com/jellyfin/jellyfin/pull/2821), [jf#2715](https://github.com/jellyfin/jellyfin/pull/2715)), with format toggles, options for toggling transcoding for HEVC and 10-bit VP9 videos off for older GPUs, providing support for the VP8/VP9 QSV and NVDEC decoders in FFmpeg 4.3, better support for UTF-16 subtitles and a host of other improvements.
```

Comment:
we could also make mention of hwaccel for mac's but it's still really beta with needing lots of testing. not sure if our current ffmpeg supports it, Anthony might have more feedback

Reply:
@anthonylavado Any info on this?

Reply:
I don't think I can reliably test it because of the age of my Mac. Technically it should be there.